### PR TITLE
Extract hdp3 tests to separate suite

### PR DESCRIPTION
**Pull request by kokosing**
Extract hdp3 tests to separate suite

### CODE REVIEW

**Comment by kokosing** [comment]
Fixes #3121

File: presto-product-tests/bin/product-tests-suite-8-non-generic.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```
Comment (kokosing) [inline presto-product-tests/bin/product-tests-suite-8-non-generic.sh]: I would prefer to name this file `product-tests-suite-hdp3.sh`, but this against the convention.

WDYT @findepi @electrum ?

Comment (findepi) [reply presto-product-tests/bin/product-tests-suite-8-non-generic.sh]: please don't change the convention

File: .github/workflows/product-tests.yml [hunk 2]
```diff
@@ -26,6 +26,7 @@ jobs:
           suite-5,
           suite-6-non-generic,
           suite-7-non-generic,
+          suite-8-non-generic,
```
Comment (findepi) [inline .github/workflows/product-tests.yml]: add this to `exclude` below (twice)

**Pull request merged**